views/default/core/settings/statistics/numentities.php

```diff
@@ -13,25 +13,25 @@
 	return;
 }
 
-$rows = '';
-
-foreach ($entity_stats as $k => $entry) {
-	foreach ($entry as $a => $b) {
-		if (elgg_language_key_exists("collection:{$k}:{$a}")) {
-			$a = elgg_echo("collection:{$k}:{$a}");
-		} else {
-			$a = "$k $a";
+$rows = [];
```

Comment:
can't we have a helper view for entity stats (as you also use it on the admin user details)